### ISSUE TITLE
the retrieval loss doesn't converge well

### ISSUE THREAD

**Issue by qq283215389** [ruotianluo/DiscCaptioning#11]
Hello, luo
when I pretrain the VSEFCmodel,  the vse_loss doesn't converge well , just around 51.2. is there some mistakes in my experiments, how about your vse_loss when you pretrain VSEFCmodel?

**Comment by ruotianluo** [comment]
thats very common in the first several epochs. Try training it a little bit longer. Or just restart the training.

**Comment by qq283215389** [comment]
ok, thanks a lot, for another VSE model(VSEAttModel) and "pair loss" , whose result isn't shown in your paper "Discriminability objective for training descriptive captions" in CVPR 2018?

**Comment by ruotianluo** [comment]
Pair loss is worse and vseattmodel gives worse result too.

**Comment by qq283215389** [comment]
thanks！if the retrieval model perform better（like the paper“Stacked Cross Attention for Image-Text Matching”），can we get a better result for captioning model？

**Comment by ruotianluo** [comment]
I think it's very likely.

**Comment by qq283215389** [comment]
hello,luo
It's my result of pre-training retrieval model after i run  “run_fc_con.sh”, there is still a difference with your result presented in your paper for the retrieval model.
Result: 
Average i2t Recall: 53.9
Image to text: 29.9 59.2 72.6 4.0 19.6
Average t2i Recall: 42.3
Text to image: 20.6 46.5 59.8 7.0 40.8

**Comment by ruotianluo** [comment]
Did you download my pretrained model? Does it perform better and the same as what's reported in the paper?
https://drive.google.com/open?id=1oQ_O-O2KoSQv1xdBPKaIOGt-VW0gS-42
These are my training curves, to give you a hint.

**Comment by qq283215389** [comment]
i might get the problem，i have used  the size of 7*7  for coco fc features, i think u have used 14*14 for coco fc features?

**Comment by qq283215389** [comment]
i might get the problem，i have used the size of 7x7 for coco fc features, i think u have used 14x14 for coco fc features?

**Comment by ruotianluo** [comment]
fc feature doest have spatial dimensions, it's a vector

**Comment by qq283215389** [comment]
I found other paper use Karpathy'split for COCO, your paper use rama's split, whose test data are the same? why you can compare your result with the result in self-critical?

**Comment by ruotianluo** [comment]
the splits are different. The self critical one is my implementation on Rama's split. Using Rama split I'd because we need to compare ours to Rama's result.
